### PR TITLE
Removes some genetic defects

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
They seem to be highly upopular and clog up the blocks or whatever I dunno how genetics works.

### CODE REVIEW

**Comment by Xhuis** [comment]
:+1:

**Comment by KorPhaeron** [comment]
Strange, it compiles locally, my master branch must be out of date.

**Comment by Razharas** [comment]
Removed some mutations - decrease the amount of dna blocks accordingly

**Comment by PKPenguin321** [comment]
genetics will be *thaaat* much more tolerable with this
:+1:

**Comment by Incoming5643** [comment]
The only thing having four versions of torrettes ever did was make perma stunning people that much easier.

**Comment by Iamgoofball** [comment]
hey faggots

you realize what you could do INSTEAD

is keep the disabilities

# and just lower the amount of blocks, fixing the problem

File: code/__DEFINES/genetics.dm [hunk 1]
```diff
@@ -63,7 +60,7 @@
 #define DNA_FACIAL_HAIR_STYLE_BLOCK	6
 #define DNA_HAIR_STYLE_BLOCK		7
 
-#define DNA_STRUC_ENZYMES_BLOCKS	23
+#define DNA_STRUC_ENZYMES_BLOCKS	20
```
Comment (Iamgoofball) [inline code/__DEFINES/genetics.dm]: just keep the mutations in but lower the amount of blocks down to whatever

File: code/game/objects/items/weapons/dna_injector.dm [hunk 2]
```diff
@@ -292,30 +292,6 @@
 		..()
 		add_mutations.Add(mutations_list[WACKY])
 
-/obj/item/weapon/dnainjector/antimute
-	name = "\improper DNA injector (Anti-Mute)"
-	New()
-		..()
-		remove_mutations.Add(mutations_list[MUT_MUTE])
-
-/obj/item/weapon/dnainjector/mutemut
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/dna_injector.dm]: you fucked up

File: code/__DEFINES/genetics.dm (hunk 1 above)
Comment (phil235) [inline code/__DEFINES/genetics.dm]: You removed 4 mutations, so 23-4=19

**Comment by phil235** [comment]
@Iamgoofball You mean make these mutations NON SCANNABLE (like laser eyes) so they can't be acquired via SE manipulation? (but keeping the injector for admin to spawn)

**Comment by Dorsisdwarf** [comment]
He just wants his memes to stick

**Comment by Shapsy** [comment]
You borked it

**Comment by Core0verload** [comment]
:+1: These mutations were shit.

**Comment by Niknakflak** [comment]
Goofball is just mad that someone is removing his shit meme disabilities.  :+1:

**Comment by freerealestate** [comment]
:+1: these were annoying and goon-level

**Comment by PKPenguin321** [comment]
DISREGARD GOOF, KILL THE CODEBABY

File: code/__DEFINES/genetics.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/__DEFINES/genetics.dm]: Yeah but some guy added dwarfism so it's 20

**Comment by Durkel** [comment]
As the one guy who actually does genetics, this is good change.

**Comment by Miauw62** [comment]
yes you could just lower the amount of blocks and then we would get dumbass fucking meme mutations instead of mutations that are actually somewhat interesting gameplay-wise.

**Comment by Kromgar** [comment]
:+1: Goofball is just mad that his shit code is getting removed

**Comment by Kromgar** [comment]
Is it possible to reduce the amount of blocks and make the defects random?

**Comment by MrPerson** [comment]
:+1: I shoulda done this myself.

Genetics is SO shitty with this many blocks and the speech modifiers are really stupid

**Comment by Iamgoofball** [comment]
Would probably be best to re-add these as chemicals then.

**Comment by PKPenguin321** [comment]
>Would probably be best to re-add these as chemicals then.

actually not a bad idea

**Comment by Miauw62** [comment]
actually a really bad idea

**Comment by Niknakflak** [comment]
>my memes suck in genetics
I KNOW, I'LL PUT THEM SOMEWHERE ELSE!
>Now they are going to be chems, oh great

**Comment by PKPenguin321** [comment]
>implying you even have to use the chems

**Comment by Razharas** [comment]
@KorPhaeron

**Comment by Xhuis** [comment]
rip ;-;7

**Comment by KorPhaeron** [comment]
Been busy, will get to it in a few days

**Comment by Xhuis** [comment]
thanks fam

**Comment by MrPerson** [comment]
[few days intensifies]

**Comment by WJohn** [comment]
RIP.

**Pull request closed without merging**